libcontainer/rootfs_linux.go

```diff
@@ -174,6 +175,10 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 		if err != nil {
 			return err
 		}
+		flags := m.Flags
+		if flags&syscall.MS_RDONLY != 0 {
+			logrus.Warn("Cgroups are not mounted as read-only, container might affect host's resources.")
```

Comment:
No, I think we don't, we just break our output. I think I'm not ready for introducing logging to child, maybe only if there will be `logrus.SetOutput(ioutil.Discard)` by default.